### DISCUSSION TITLE
Add ok/close button

### DISCUSSION THREAD

**Discussion by juanesf** [meerk40t/meerk40t#1961] [Ideas]
Today I was testing the new delete button, and I tried to draw a picture with the vector button. The problem is that when "finishing" the vector I don't have how to press the "enter" key to end or close the design.

[https://we.tl/t-NCV6mXZal5](url)

**Comment by tatarize** [comment]
Well, vector is a bit iffy as a tool. It seems like maybe there's easier methods to close off the vector, I think jpirnay adjusted it a bit, but initially it was pretty bad. Just a random tool I worked up in half an hour or so. 

Looks like there's angle snaps when you hold down alt.
Leftdown sets down a position.
Right mouse aborts the position.
Move lets you place the control point.
Left up, adds the control point at that time.
Double-click (left) ends the tool.
Mouse-lost or `escape` deactivates the tool.

end_tool() is what actually adds the path. So that would seem like it would happen with a double-press.
